models/AppModel.js

```diff
@@ -0,0 +1,14 @@
+var AppModel = Backbone.Model.extend({
+  defaults() {
+    return {
+      videos: new VideosCollection(),
+
+      current_query: null
+    }
+  },
+
+  changeCurrentQuery(query){
+    this.set('current_query', query);
+    this.get('videos').newSearch(query);
```

Comment:
You could also tie this to a change event so that it happens whenever the query is updated.